packages/@ember/-internals/glimmer/tests/integration/components/tracked-test.js

```diff
@@ -485,6 +485,55 @@ if (EMBER_METAL_TRACKED_PROPERTIES) {
           assert.equal(outerRenderCount, 2, 'outer component updates based on context');
           assert.equal(innerRenderCount, 3, 'inner component updates based on outer component');
         }
+
+        '@test computed properties computed properties can depend on args'() {
```

Comment:
The phrasing here seems odd, is it intentional?

Reply:
lol, definitely not, will fix 😛